packages/ui/src/components/vuestic-components/va-option-list/VaOptionList.demo.vue

```diff
@@ -252,6 +375,13 @@ export default {
         selectedObject: undefined,
         withDefault: undefined,
       },
+      baseSwitchData: {
+        options: ['test switch', 'test switch 1', 'test switch 2'],
+        // bug when selected is true
```

Comment:
I mean like with red border. So it's visually clear.
Also any reason not to fix it? Can you create an issue if it's hard to fix?